ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -930,7 +930,80 @@ FCU_DFSS_DFVSC_DXRC_HTRC_RHC_RHDHC_RTC:
   - RHC
   - RHDHC
   - RTC
-  
+
+FCU_SS_ZTC:
+  description: "Single split unit with zone air temperature control"
+  is_canonical: true
+  implements:
+  - FCU
+  - ZTC
+  - SS
+  opt_uses:
+  - failed_alarm
+  - lost_power_alarm
+  - schedule_run_command
+
+FCU_DFSS_CHWVM_ZTC:
+  description: "Chilled water FCU with zone temperature control and chilled water valve control"
+  is_canonical: true
+  implements:
+  - FCU
+  - DFSS
+  - CHWVM
+  - ZTC
+  opt_uses:
+  - schedule_run_command
+
+FCU_DFSS_ZTC_CWVC:
+  description: "Chilled water FCU with binary mode (open/closed) and zone temperature control"
+  is_canonical: true
+  implements:
+  - FCU
+  - DFSS
+  - ZTC
+  uses:
+  - chilled_water_valve_command
```

Comment:
since this is binary, it probably requires a new type; we have precedent for a similar type, so your new type would look something like this:

```
CHWBZC:
  description: "Two-position chilled water valve (open/closed) for zone temp control."
  is_abstract: true
  implements:
  - OPERATIONAL
  uses:
  - chilled_water_valve_command
  - zone_air_temperature_sensor
  - zone_air_temperature_setpoint
```